### ISSUE TITLE
Switching back to netrw in insert mode causes modifiable errors

### ISSUE THREAD

**Issue by mqudsi** [neovim/neovim#7286]
Under `NVIM v0.2.1-867-gd2cbc31`, the following causes errors about the buffer not being modifiable to show up:

```
nvim -u NORC
:vsp
:set mouse=a
#click in right pane
:Explore
#click in left pane
i
#click in right pane
#click in left pane
```

Expected: back in left pane in insert mode
Actual: error about modifiable being off, in insert mode, in right pane

**Comment by justinmk** [comment]
Does it happen in Vim 8.x with the same settings?

**Comment by mqudsi** [comment]
No, but note that in vim control of the mouse is given up in insert mode. It's not possible to switch to another window by clicking in it while in insert mode.

**Comment by jamessan** [comment]
> It's not possible to switch to another window by clicking in it while in insert mode.

That's not true.  You just need to use the same settings as in nvim.  If you have `mouse=a` or `mouse=i`, then you can switch windows while in insert mode.

**Comment by mqudsi** [comment]
I don't know, for me, with `mouse=a` under conhost with `VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Nov 24 2016 16:44:48)`, `:set mouse=a` does not let me switch between windows while in insert mode.

**Comment by jamessan** [comment]
Then that's probably a problem with either vim not recognizing the control sequences from conhost or conhost not supporting mouse interaction.  Take the terminal out of it and use gvim.
